src/App.js

```diff
@@ -186,7 +186,7 @@ class App extends Component {
     if (this.state.error) {
       return (
         <div>
-          <h1>{ this.state.error }.</h1>
```

Comment:
The period is here by intention, as Last.FM API errors are coming without it.

Reply:
Wouldn't the better solution be to append the dot the the string before printing rather than enforcing all error messages to end with a dot?